docs/domain/ids.md

```diff
@@ -2,10 +2,10 @@
 
 The domain part of the library supports a set of generic IDs:
 
-1. [`Id<T>`](../../src/Domain/LeanCode.DomainModels/Model/Id.cs),
-2. [`IId<T>`](../../src/Domain/LeanCode.DomainModels/Model/Id.cs),
-3. [`LId<T>`](../../src/Domain/LeanCode.DomainModels/Model/Id.cs),
-4. [`SId<T>`](../../src/Domain/LeanCode.DomainModels/Model/SId.cs).
+1. [`Id<T>`],
```

Comment:
Outside of the scope, but these are deprecated, docs should be adjusted.

Reply:
Added